src/platform/mock-local-home-platform.ts

```diff
@@ -106,39 +37,18 @@ export class MockLocalHomePlatform implements MockUDPListener {
     return this.localDeviceIds;
   }
 
-  private setupUDP() {
-    this.udpScanConfigs.forEach(udpScanConfig => {
-      this.mockNetwork!.registerUDPListener(
-        this,
-        udpScanConfig.listenPort,
-        udpScanConfig.broadcastAddress
-      );
-    });
-  }
-
-  private sendUDPBroadcast(scanConfig: UDPScanConfig) {
-    const packetBuffer = Buffer.from(scanConfig.discoveryPacket, 'hex');
-    this.mockNetwork!.sendUDPMessage(
-      packetBuffer,
-      scanConfig.broadcastPort,
-      scanConfig.broadcastAddress,
-      scanConfig.listenPort,
-      scanConfig.broadcastAddress
-    );
-  }
-
-  // Establish fulfillment path using app code
-  async onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): Promise<void> {
-    if (this.app === undefined) {
-      throw new Error('Cannot trigger IdentifyRequest: App was undefined');
+  /**
+   * Takes a `discoveryBuffer` and passes it to the fulfillment app in an `IdentifyRequest`
+   * @param discoveryBuffer  The buffer to be included in the `IdentifyRequest` scan data
+   * @returns  The next localDeviceId registered to the Local Home Platform
+   */
+  public async triggerIdentify(discoveryBuffer: Buffer): Promise<string> {
```

Comment:
curious if we could return `this` here to allow chaining it with assertion? (see https://github.com/kozhevnikov/proxymise for a way to enable fluent async interface).

Reply:
Looks interesting, I opened an issue for it (#35)